zerver/lib/message.py

```diff
@@ -320,6 +322,7 @@ def render_markdown(message, content, realm=None, realm_alert_words=None, messag
             if user_id in message_user_ids:
                 possible_words.update(set(words))
 
+    available_slash_commands = set(get_slash_commands_by_realm(realm, is_active=True))
```

Comment:
Nit: Can you please pass this as a method parameter (e.g. `realm_slash_commands`) so that it's consistent with how `realm_alert_words` is handled?

(For example, see https://github.com/zulip/zulip/blob/master/zerver/lib/actions.py#L692)